README.md

```diff
@@ -50,30 +63,116 @@ Please refer to these defined terms throughout the documentation to ensure a con
 
 ## Implementation Details
 
-The implementation of Nitrokey WebCrypt incorporates the following design considerations:
+The implementation of Nitrokey WebSmartCard incorporates the following design considerations:
 
-1. **Inspiration from OnlyKey's WebCrypt**: The solution draws inspiration from OnlyKey's WebCrypt proof-of-concept,
+1. **Inspiration from OnlyKey's WebSmartCard**: The solution draws inspiration from OnlyKey's WebSmartCard proof-of-concept,
```

Comment:
I don't think this should be renamed here.

Reply:
Good catch, corrected